api/schemas/flow-segment-bulk-failure.json

```diff
@@ -0,0 +1,32 @@
+{
+    "title": "Failed segments",
+    "description": "List of segments that have failed to register",
+    "type": "object",
+    "required": [
+        "failed_segments"
+    ],
+    "properties": {
+        "failed_segments": {
+            "description": "The object ID of the segment which has failed to register with the TAMS API",
+            "type": "object",
```

Comment:
See the actions failure - this should be an array with object items.

Reply:
I note that the linter accepted your change but note that in addition to changing the type to `array` it needs an `items` with the `object` type. See for example the [pre property](https://github.com/bbc/tams/blob/45a850c4f379f62903dbe57c18e08ddaaa260180/api/schemas/flow-storage.json#L9)